freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -143,6 +143,11 @@ def _get_bundles_to_rebuild(self):
         log.debug("Getting NVR mapping of original images to rebuilt images")
         nvrs_mapping = self._create_original_to_rebuilt_nvrs_map()
 
+        if (isinstance(self.event, ManualBundleRebuildEvent) and 
+            self.event.requester_metadata_json.get("bundle_related_image_overrides", False)):
```

Comment:
Just check `self.event.requester_metadata_json.get("bundle_related_image_overrides", False)` is good enough.